KamerConnect/Utils/EnumUtils.cs

```diff
@@ -11,4 +13,10 @@ public static T Validate<T>(string value) where T : struct
 
         throw new ArgumentException($"Invalid value '{value}' for enum {typeof(T).Name}");
     }
+    public static string GetEnumDescription(this Enum value)
```

Comment:
Niek heeft de GetAttribute aangemaakt en heeft description displayName genoemd. De Enum utils vind ik een mooiere plek aangezien we het alleen voor utils gebruiken, maar dit moet je dan wel even concistent maken. DisplayName vind ik wel een betere naam.